### PR TITLE
Oak's Turtle and basic Python Exercise

### PR DESCRIPTION
**Pull request by Blucka12**
@eah13 @gerbal This is the submission of my turtle exercise, which is admittedly crude and in need of more love. Also included is my python exercise, which is in development and should be ignored for now. Thanks!

### CODE REVIEW

**Comment by Blucka12** [comment]
Ok, @eah13  @gerbal  , the .md entitled "Oak's Second Assignment" is the Assignment due tonight. Hopefully, I can get clear on how to complete this task without causing a mess next time. What do I need to do differently to edit the Pull request I was working on?

**Comment by gerbal** [comment]
@Blucka12 a couple of things
First you can combine your two python posts into one, it will make our lives a little easier.  
Second, in your turtle post should be an actual post. Make sure to include the header, name the file consistently, and include a picture of the output.  
Third, put your code in `````code blocks, it's easier to read and MarkDown won't try to format it like normal text.

**Comment by Blucka12** [comment]
Ok @gerbal , I think I deleted the stuff I didn't want to be part of the pull request. I didn't do the pic of the output bit yet and I still have to fix the turtle post. I'll get to that soon.

**Comment by eah13** [comment]
@Blucka12 Did you preview your site when submitting it?  Next time make sure to include a link. 

When I visit http://blucka12.github.io/spring2014/ I see both of these posts missing their authors and with incorrect dates.  @gerbal's got the right idea on how to fix the author and the dates are part of the filename.

File: _posts/2014-01-17-blucka12turtle.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+format: post
```
Comment (gerbal) [inline _posts/2014-01-17-blucka12turtle.md]: This should be `layout: post`

**Comment by eah13** [comment]
@Blucka12 see if you can get this post to have the right date (this will mean changing the filename) and the code block to show up correctly before we chat this PM.

**Comment by Blucka12** [comment]
I'm unclear on what date should be on the post. The "Turtle Assignment" was posted on the 17th before midnight, but the other stuff came later, methinks.

When I "preview" the code block, it appears the way I feel others show up, but after the commit, it seems to get wonky...not sure what I'm getting wrong, there...

**Comment by Blucka12** [comment]
I'm trying to delete "OakPythonExercise.md", but am not able to get to that option...

That was just practice we were doing in class that I mistakenly made into the pull request...

**Comment by gerbal** [comment]
@Blucka12 you already deleted it in commit b3159a6

**Pull request merged**